api/v1/kubtest.yaml

```diff
@@ -442,27 +448,10 @@ components:
           description: 'test execution end time'
           format: date-time
 
-    Execution:
+    Result:
```

Comment:
lets call this ExecutionResult (there could be other types of results down the line..)

Reply:
ok makes sense gimme 5 mins :D